tools/hybrid-quickstart/hybrid13/steps.sh

```diff
@@ -180,7 +182,7 @@ create_apigee_org() {
     "https://apigee.googleapis.com/v1/organizations?parent=projects/$PROJECT_ID"
 
     echo -n "⏳ Waiting for Apigeectl Org Creation "
-    wait_for_ready "0" "curl --silent -H \"Authorization: Bearer $(token)\" -H \"Content-Type: application/json\" https://apigee.googleapis.com/v1/organizations/$PROJECT_ID | grep \"subscriptionType\" > /dev/null  2>&1; echo \$?" "Organization $PROJECT_ID is created."
+    wait_for_ready "0" "curl --silent -H \"Authorization: Bearer $(token)\" -H \"Content-Type: application/json\" https://apigee.googleapis.com/v1/organizations/$PROJECT_ID | grep -q \"subscriptionType\"; echo \$?" "Organization $PROJECT_ID is created."
```

Comment:
is this coming from shellcheck? Which version has better compatibility with all types of shells?

Reply:
Nope not coming from shellcheck just simpler and easier on the eye. 

`-q` is supported in both:
* GNU grep https://www.gnu.org/software/grep/manual/grep.html 
* BSD grep https://www.freebsd.org/cgi/man.cgi?query=grep&sektion=&n=1